tasks.py

```diff
@@ -4,19 +4,12 @@
 import contextlib
 import glob
 import os
+import shutil
```

Comment:
This whole file can be replaced with a super simple version of it using `compas_invocations` (again refer to this [pull request](https://github.com/compas-dev/compas/pull/1088/files)).